src/config/entities/configuration.ts

```diff
@@ -81,6 +81,9 @@ export default () => ({
       'https://api.coingecko.com/api/v3',
     apiKey: process.env.PRICES_PROVIDER_API_KEY,
     pricesTtlSeconds: parseInt(process.env.PRICES_TTL_SECONDS ?? `${300}`),
+    notFoundPriceTtlSeconds: parseInt(
+      process.env.NOT_FOUND_PRICE_TTL_SECONDS ?? `${259200}`,
```

Comment:
I would say that having `${70 * 60 * 60}` is more readable.

Reply:
Changed in [7538cd9](https://github.com/safe-global/safe-client-gateway/pull/828/commits/7538cd923d47b7eaefbfde53d1482a36af131894)